### PR TITLE
Add close-to assertions

### PR DESCRIPTION
**Pull request by SandeepJoelFreshdesk**

### CODE REVIEW

File: test-support/assertions/close-to.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export default function(context, actual, target, delta, message) {
```
Comment (shibulijack-fd) [inline test-support/assertions/close-to.js]: Can you rename target to expected? Just to make it obvious. `target` works for cases where the same param accepts multiple type values.

File: test-support/assertions/close-to.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  message = message || `${actual} should be have a delta of ${delta}"`;
+  let result = false;
+  if (
+    actual && typeof actual === 'number' &&
+    target && typeof target === 'number' &&
+    delta && typeof delta === 'number'
+  ) {
+    result = target == Math.abs(actual - delta);
```
Comment (shibulijack-fd) [inline test-support/assertions/close-to.js]: Strict equals?

Comment (SandeepJoelFreshdesk) [reply test-support/assertions/close-to.js]: done

File: test-support/assertions/close-to.js (hunk 1 above)
Comment (SandeepJoelFreshdesk) [reply test-support/assertions/close-to.js]: done

**Pull request merged**